panoptes_aggregation/tests/panoptes_tests/test_userify.py

```diff
@@ -0,0 +1,134 @@
+from importlib import import_module
+
+from unittest.mock import MagicMock, PropertyMock, Mock
+from nose.tools import assert_equals
+panoptes = import_module('panoptes_aggregation.panoptes', __name__).panoptes_testing
+from panoptes_client import Panoptes, User
+
```

Comment:
The `env` variables for `FLASK_PANOPTES_ID` and `FLASK_PANOPTES_SECRET` should be set to some default values in case they are not already defined:

```python
from os import environ

environ.setdefault('FLASK_PANOPTES_ID', 'TEST')
environ.setdefault('FLASK_PANOPTES_SECRET', 'TEST')
```

Reply:
got it!

Reply:
Although fwiw I stub out calls to the actual API so that no HTTP traffic should occur.